src/types.ts

```diff
@@ -4,11 +4,14 @@ import { TouchEventInput } from "./eventInput/TouchEventInput";
 import { PointerEventInput } from "./eventInput/PointerEventInput";
 import { TouchMouseEventInput } from "./eventInput/TouchMouseEventInput";
 import { InputType } from "./inputType/InputType";
+import { EasingManager } from "./animation/EasingManager";
 
 export type ObjectInterface<T = any> = Record<string | number, T>;
 
 export type InputEventType = PointerEvent | MouseEvent | TouchEvent;
 
+export type AnimationManagerType = EasingManager;
```

Comment:
Is there any specific reason that this is needed? Why don't you just use `AnimationManager` for its type?

Reply:
Assuming that all Animation types to be added later inherit AnimationManager, It seems to be bad to separate Animation type into other type. Therefore, it will be removed.